ipynbs/Chasing_Platypus_via_MG-RAST.ipynb

```diff
@@ -25,6 +25,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "<b>Before you continue: </b>This step is extremelly slow (~24 hrs.) and network intensive. Thus to facilitate this step you could download the files from April 22, 2015 [ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz](ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz). If you place them in the same folder that you have this IPython notebook the code will simply download the latests samples.\n",
```

Comment:
notebook -> Notebook

btw, since you're modding this, possible to add a license mention in the top cell?